src/core/server/legacy/plugins/find_legacy_plugin_specs.ts

```diff
@@ -25,6 +25,7 @@ import {
   // @ts-ignore
 } from '../../../../legacy/plugin_discovery/find_plugin_specs.js';
 import { LoggerFactory } from '../../logging';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
interesting that it wasn't detected by the rule before. need to take a closer look